scripts/release_prep.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -x
+
+# pre-requisites: install github CLI
+# - github documentation: https://github.com/cli/cli#installation
+# - github is remote 'origin'
+
+git fetch --all --tags
+git reset --hard
+git pull
+git checkout develop
+git reset --hard origin/develop
+
+short_hash=$(git rev-parse --short HEAD)
+temp_release_branch=temp_${short_hash}
+
+git chekcout -b ${temp_release_branch}
```

Comment:
typo!!!
```suggestion
git checkout -b ${temp_release_branch}
```

Reply:
OMG Thanks! 

LOL, this is a typo I always do, so I have a local alias and it prevented me from see from seeing it 😅 

```[alias]
	chekcout = checkout
```